src/main/resources/swagger.api/schemas/entityType.json

```diff
@@ -91,6 +91,14 @@
           "type": "string"
         },
         "default": []
+      },
+      "additionalEcsConditions": {
+        "description": "Additional conditions to be included in the WHERE clause of this entity type in ECS environments",
+        "type": "array",
+        "items": {
+          "type": "string"
+        },
+        "default": []
```

Comment:
I made this an array in case we find some other condition that we need for ECS environments later (not that I'm expecting to, just figured it couldn't hurt).